### PR TITLE
Remove projectSummary from response (#1021)

### PR DESCRIPTION
**Pull request by danielsotirhos**

### CODE REVIEW

File: test/service/test_repository_projects.py [hunk 1]
```diff
@@ -53,7 +53,6 @@ def assert_file_type_summaries(hit):
             assert_file_type_summaries(hit)
             self.assertTrue('projects' in hit)
             self.assertTrue('specimens' in hit)
-            self.assertTrue('projectSummary' in hit)
```
Comment (jessebrennan) [inline test/service/test_repository_projects.py]: Why not make assertFalse instead, to test that it is truly removed?

File: test/service/test_repository_projects.py [hunk 2]
```diff
@@ -82,7 +81,6 @@ def _test_detail_response(self, uuid):
         self.assertTrue('entryId' in hit)
         self.assertTrue('projects' in hit)
         self.assertTrue('specimens' in hit)
-        self.assertTrue('projectSummary' in hit)
```
Comment (jessebrennan) [inline test/service/test_repository_projects.py]: Same as above.

File: test/service/test_repository_specimen.py [hunk 3]
```diff
@@ -46,7 +46,6 @@ def assert_file_type_summaries(hit):
             self.assertTrue('donorOrganisms' in hit)
             self.assertTrue('organoids' in hit)
             self.assertTrue('cellSuspensions' in hit)
-            self.assertFalse('projectSummary' in hit)
```
Comment (jessebrennan) [inline test/service/test_repository_specimen.py]: Is there any harm in leaving this line in? Maybe a comment could explain that it's been deprecated.

**Comment by NoopDog** [comment]
Please deploy to dev.

File: test/service/test_repository_projects.py (hunk 1 above)
Comment (hannes-ucsc) [reply test/service/test_repository_projects.py]: there is also `assertIn` which would yield a better failure message.

Comment (hannes-ucsc) [reply test/service/test_repository_projects.py]: And yes, inverting rather than removing makes sense. So use `assertNotIn`

Comment (danielsotirhos) [reply test/service/test_repository_projects.py]: Implemented check for non-existence of the 'projectSummary' object with `assertNotIn`

**Pull request merged**